euphoria-core/src/main/java/cz/seznam/euphoria/core/client/operator/Join.java

```diff
@@ -442,7 +442,7 @@ public boolean isOuter() {
         OUT, JoinState, W> reduce;
 
     name = getName() + "::ReduceStateByKey";
-    reduce = new ReduceStateByKey<>(
+    reduce = new ReduceStateByKey(
```

Comment:
no problem. just keep it here. that's fine with me.

is there any chance an upgrade to eclipse would solve the issue (given that `javac` has no problem compiling it i'd expect there's a particular issue with the eclipse compiler?) it seems worth being traced back to the root-cause and documented (maybe in the wiki)